src/ckb-daemon/usb_mac.c

```diff
@@ -271,6 +271,67 @@ static void pipecomplete(void* refcon, IOReturn result, void* arg0){
     (*handle)->ReadPipeAsync(handle, ctx->pipe, buffer, ctx->maxsize, pipecomplete, ctx);
 }
 
+CFMachPortRef event_tap;
```

Comment:
This declaration should be on top of the file. Also there is a documentation missing: What does this reference store?

Also, is it a static variable inside this file?